apps/web/src/components/Icon/Icon.tsx

```diff
@@ -7,8 +7,12 @@ export enum iconColor {
 }
 
 interface Props {
+  /** What should be displayed? */
   children: React.ReactElement;
+  /** The color of the Icon */
   colorProp?: iconColor;
+  /** What should be displayed? */
```

Comment:
NIT: This is the same comment as line 10

Also, the JSDocs don't _always_ need to be phrased as a question, especially if it doesn't make the comment clearer. Its a thing you'll get a feel for with time.